### PR TITLE
commit the MoveFilesToTrashCommand code

### PR DESCRIPTION
**Pull request by sorena-dev**
Hi Austin, 

I'm sending the updated code with MoveFilesToTrashCommand class. I'm a beginner dev and would be very happy to have your feedback. 

Best Regards,
Sorena

### CODE REVIEW

**Comment by locaaustin** [comment]
Thanks for contributing. I will take a look at it tonight and provide any feedback.

File: pythonfilemover/main.py [hunk 1]
```diff
@@ -212,9 +217,53 @@ def process_file_move(self):
         shutil.move(self.directory_from, self.directory_to)
         print(f"SUCCESS: The Folder has been moved from {self.directory_from} to {self.directory_to}")
 
+class MoveFilesToTrashCommand():
+    def __init__(self, dir_a, dir_b): 
+        self.directory_from = None
```
Comment (locaaustin) [inline pythonfilemover/main.py]: I am not understanding setting self.directory_from to None.  The user would need to define a from directory as we want to move all files from self.directory_from to the recycle bin.

File: pythonfilemover/main.py [hunk 2]
```diff
@@ -212,9 +217,53 @@ def process_file_move(self):
...
+        self.directory_to = recycle_bin
+        self.user_extensions = []
+    
+    def process_move(self):
+        output_padding = " " * (len("Name") - len("Extension"))
+        #Obtain extensions from our Extension Process first.
+        click.echo("Please choose the obtain below to begin: ")
```
Comment (locaaustin) [inline pythonfilemover/main.py:291]: I would recommend changing the 'obtain' to 'option' below.

**Review by AustinCGomez — CHANGES_REQUESTED** [review]
Great job on working on this commit. I believe that you are almost there to what we need. I did notice two potential errors. 

First: MoveFilesToTrashCommand() would require that a directory_from is selected. I ran the program and it was not working with moving any files since that directory was left to blank. 

Second: In DefineCommand()class within def main(self) there is no click.prompt for --move-to-recycle. I would add one there for --move-to-recylce in a similar fashion to this "--move-folders - Move Entire Folders from Directory A to Directory B!.

**Comment by sorena-dev** [comment]
Hi Austin, I'm sorry I couldn't respond sooner. I decided to adopt the easier solution as I had to deal with other problems. I used send2trash module and it works. I removed dir_a and dir_b and the class MoveFilesToTrashCommand takes no argument. I have tested it on my pc, and it worked. Can you check the new pull?

**Review by AustinCGomez — APPROVED** [review]

**Comment by locaaustin** [comment]
Hi @Sorena1994 excellent work on the new solution and it does work! I'll have this pull request merged soon.

**Pull request merged**